### PR TITLE
add accountability as a key design property

### PR DESCRIPTION
**Pull request by dkg**
When is a verifier allowed to request a credential?  What hooks do the mechanisms offer to attach consequences to a verifier who requests access to credentials that they do not need?  How can issuers be held accountable?

### CODE REVIEW

**Comment by alenhorvat** [comment]
Related work: https://api-pilot.ebsi.eu/docs/specs/trust-model/credentials-presentation-and-request  (open for feedback/comments)

**Review by OR13 — APPROVED** [review]

File: charter.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ The SPICE WG's work items will not require nor forbid the use of JSON-LD. Retain
 - Availability (efficiently information processing, minimizing data in flight, reducing carbon cost for storage and transmission, and supporting offline / paper credential use cases)
 - Semantic interchangeability (ensuring terms are understood and used consistently, in a global or local context)
 - Usability and Feasibility ( The core operations must be easy to understand and use, and possible to integrate with existing platforms)
+- Accountability (what format and protocol considerations can help mitigates abusive or excessive requests for credential presentation?  what mitigates abuse by issuers?)
```
Comment (OR13) [inline charter.md]: I like this, and as one example, we might consider providing guidance on risks associated with requiring or forbidding attestation.

Comment (brentzundel) [inline charter.md]:
```suggestion
- Accountability (what format and protocol considerations can help mitigate abusive or excessive requests for credential presentation?  What mitigates abuse by issuers?)
```

**Review by brentzundel — APPROVED** [review]

File: charter.md [hunk 2]
```diff
@@ -59,6 +59,7 @@ The SPICE WG's work items will not require nor forbid the use of JSON-LD. Retain
 - Availability (efficiently information processing, minimizing data in flight, reducing carbon cost for storage and transmission, and supporting offline / paper credential use cases)
 - Semantic interchangeability (ensuring terms are understood and used consistently, in a global or local context)
 - Usability and Feasibility ( The core operations must be easy to understand and use, and possible to integrate with existing platforms)
```
Comment (henkbirkholz) [inline charter.md]:
```suggestion
- Usability and Feasibility (the core operations must be easy to understand and use, and possible to integrate with existing platforms)
```
misusing this PR to fix some close by style issues

**Review by henkbirkholz — APPROVED** [review]

**Pull request merged**